src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -136,7 +133,7 @@ public bool IsChannelReadyForInvocations()
         public async Task StartWorkerProcessAsync()
         {
             _startSubscription = _inboundWorkerEvents.Where(msg => msg.MessageType == MsgType.StartStream)
-                .Timeout(TimeSpan.FromSeconds(WorkerConstants.ProcessStartTimeoutSeconds))
+                .Timeout(_workerConfig.CountOptions.ProcessStartupTimeout)
```

Comment:
rename `CountOptions` to `processOptions` or `startUpOptions` ?

Reply:
This is an existing property. If we want to refactor/rename, that should probably be done in a separate PR.

Reply:
@pragnagopa wouldn't that also be a potential breaking change? If this options type is used from configuration and binding, different sources might be relying on this property name, which would break if it changes.

Reply:
Punting for a future discusssion.

Reply:
@fabiocav  / @brettsam  -  In json it is named correctly - https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/WorkerConstants.cs#L30 -- so not a breaking change